src/native/libs/build-local.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+# Licensed to the .NET Foundation under one or more agreements.
+# The .NET Foundation licenses this file to you under the MIT license.
+#
+
+# This script is used only for building native libraries
+# in the end-user's computer for NativeAOT purposes (static linking).
+# This file is not used during the dotnet runtime build.
+
+# Currently, only Linux is supported
+
+SHIM_SOURCE_DIR=$1/native/src
+INTERMEDIATE_OUTPUT_PATH=$2
+TARGET_LIBRARY=$3
+
+if [ -d "$SHIM_SOURCE_DIR" ]; then
+    LOCAL_SHIM_DIR="$INTERMEDIATE_OUTPUT_PATH"/libs/$TARGET_LIBRARY/build
+    mkdir -p "$LOCAL_SHIM_DIR" && cd "$LOCAL_SHIM_DIR"
+    if [ $? -ne 0 ]; then echo "build-local.sh::ERROR: Cannot use local build directory"; exit 1; fi
```

Comment:
Please use the same inline syntax as before for compat. The only change needed was replacing hardcoded name with `$TARGET_LIBRARY`.

Reply:
Thank you! I copy file from browser instead of file system